stroeer/web/section/v2/teaser_page.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package stroeer.web.section.v2;
+
+import "stroeer/web/article/v1/web_article.proto";
+import "stroeer/common/v1/primitives.proto";
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/web/section/v2;section";
+option java_multiple_files = true;
+option java_package = "de.stroeer.web.section.v2";
+
+message TeaserPage {
```

Comment:
simply use `Page` or maybe `SectionPage`? Candidates here are "home" or actual sections such as `/sport`? This would be more consistent naming where we have `Article` and `Section` available in our API?

For me, the words `containers` and `stages` do not have anything in common. Maybe we can find a different naming scheme/abstraction:

`Page` > `sections[]` > `containers[]` > `articles[]`

And maybe a more generic `settings` object reused at each level at this hierarchy, `settings = Map<String, String>`?

```json
"Page": {
  "settings": "/* section_nav, title, logo, sponsoring, seo, ... */",
  "sections": [{
    "settings": "/* container_nav, container_links, container_title, container_logo, sponsoring, ... */",
    "containers": [{
      "settings": "/* row_title, row_logo, sponsoring, ... */",
      "articles": [ "List<Article>" ]
    }]
  }
  ]
}
```